packages/cli/commands/project/dev.js

```diff
@@ -93,6 +93,9 @@ exports.handler = async options => {
   if (hasPrivateApps && hasPublicApps) {
     logger.error(i18n(`${i18nKey}.errors.invalidProjectComponents`));
     process.exit(EXIT_CODES.SUCCESS);
+  } else if (!hasPrivateApps && !hasPublicApps) {
```

Comment:
The `components` should all have a `runnable` field on them. That's what the local dev manager uses here to check if there's anything that can run locally in the project. It might make sense to leverage that field so this can scale with new runnable components more easily.

Reply:
That works too! Can move that check out of the `LocalDevManager`. The rest of `dev` is already kind of set up assuming that private/public apps are the only runnable components (and will require some updates if that changes) which is why I just went with this